README.md

```diff
@@ -1 +1,39 @@
 # toxicity-demo
+
+This is a sample application that demonstrates how to build an intelligent application using Next.js, OpenAI, and Fly.io.
+
+The app uses the OpenAI API to generate natural language responses based on the input provided by the user.
+
+## Prerequisites
+
+* Next.js
+* Fly.io account
+* OpenAI API Key
+  
+## Getting Started
+
+* Clone the repository
+
+```bash
+git clone https://github.com/GuiBibeau/toxicity-demo.git
+```
+
+* Install dependencies
+
+```bash
+npm install
+```
+
+* Replace `process.env.OPENAI_API_KEY` in `pages/api/question.js` with your OpenAI API Key.
```

Comment:
let's mentioning putting it inside the `.env` file at the root of next.js. Hard coding secrets is not the best

Reply:
Okay, willl do